src/lib.rs

```diff
@@ -61,3 +63,33 @@ unsafe impl GlobalAlloc for LibcAlloc {
         libc::realloc(ptr as *mut c_void, new_size) as *mut u8
     }
 }
+
+#[cfg(target_family = "windows")]
+mod win_crt;
+
+#[cfg(target_family = "windows")]
+unsafe impl GlobalAlloc for LibcAlloc {
+    #[inline]
+    unsafe fn alloc(&self, layout: Layout) -> *mut u8 {
+        return win_crt::_aligned_malloc(layout.size(), layout.align()) as *mut u8;
```

Comment:
```suggestion
        win_crt::_aligned_malloc(layout.size(), layout.align()) as *mut u8
```

Reply:
I am not a big fan of this kind of omitted returned statement. It makes source code less readeable. If you insist, I would change it.

Reply:
I'm not sure how much Rust you've done, but it's pretty standard form to omit the return statement. After all, Rust is an expression-oriented language, and function bodies are just an expression.

On a related note, consider using [clippy](https://github.com/rust-lang/rust-clippy#usage) to lint against these common Rust idioms. In this case, the relevant lint would be [`clippy::needless_return`](https://rust-lang.github.io/rust-clippy/master/index.html#needless_return).